handlers/facebookapp/facebookapp.go

```diff
@@ -142,6 +142,7 @@ type moPayload struct {
 				IsEcho      bool   `json:"is_echo"`
 				MID         string `json:"mid"`
 				Text        string `json:"text"`
+				IsDeleted   string `json:"is_deleted"`
```

Comment:
is this definitely correct according to their docs? I notice above that `is_echo` is an actual `bool`? Can go unmarshalling convert `"true"` to `true`?

Reply:
There they send a string as shown on the webhook events example

https://developers.facebook.com/docs/messenger-platform/instagram/features/webhook#webhook-events

Reply:
Well that's some impressively messy API design on their part